packages/babel-plugin-transform-react-constant-elements/test/fixtures/constant-elements/append-to-end-when-declared-in-scope-3/expected.js

```diff
@@ -0,0 +1,20 @@
+var _ref2 = <div>child</div>;
+
+var _ref3 = <p>Parent</p>;
+
+(function () {
+  var _ref = <div>
+          {_ref3}
+          <AppItem />
```

Comment:
It is, didn't look over the test too well apparently. Needs to have the same behaviour as the top level one.